tests/Oryx.BuildImage.Tests/Python/PythonPackageTest.cs

```diff
@@ -26,7 +26,7 @@ public PythonPackageTest(ITestOutputHelper output) : base(output)
 
         public static IEnumerable<object[]> PythonPackageExamples => new object[][]
         {
-            new object[] { "boto3", "1.14.44",
+            new object[] { "boto3", "1.21.20",
```

Comment:
nit: would you mind pulling the latest changes from `main`? This change should then go away.